examples/3d-tiles/app.js

```diff
@@ -70,13 +90,39 @@ export default class App extends PureComponent {
 
   async _loadSelectedTileset(category, name) {
     const {examplesByCategory} = this.state;
-    const selectedExample = examplesByCategory[category].examples[name];
-    const tilesetUrl = `${DATA_URI}/${selectedExample.path}/${selectedExample.tileset}`;
-    let tilesetJson = null;
-    if (selectedExample.tileset) {
-      tilesetJson = await load(tilesetUrl);
+
+    let tilesetUrl;
+    let layerProps = {};
+    if (category === 'additional') {
+      layerProps = ADDITIONAL_EXAMPLES.examples[name];
+      this.setState({
+        viewState: {
+          ...this.state.viewState,
+          longitude: 144.97212,
```

Comment:
Do we still need these constants here? They are also on line 28.

Use constants for these values?